package.json

```diff
@@ -1,19 +1,31 @@
 {
+  "dependencies": {
+    "bootstrap-icons": "^1.10.5",
+    "vue-book-reader": "^1.0.7"
+  },
   "devDependencies": {
     "@babel/core": "^7.17.0",
     "@babel/preset-env": "^7.16.0",
     "@hotwired/stimulus": "^3.0",
+    "@rushstack/eslint-patch": "^1.8.0",
     "@symfony/stimulus-bridge": "^3.2",
     "@symfony/ux-autocomplete": "file:vendor/symfony/ux-autocomplete/assets",
     "@symfony/ux-live-component": "file:vendor/symfony/ux-live-component/assets",
     "@symfony/webpack-encore": "^4.6",
+    "@vitejs/plugin-vue": "^5.0.5",
+    "@vue/eslint-config-prettier": "^9.0.0",
     "autoprefixer": "^10.4.14",
     "bootstrap": "^5.3.1",
     "core-js": "^3.23.0",
+    "eslint": "^8.57.0",
+    "eslint-plugin-vue": "^9.23.0",
+    "prettier": "^3.2.5",
     "regenerator-runtime": "^0.13.9",
     "sass": "^1.62.0",
     "sass-loader": "^13.2.2",
     "tom-select": "^2.2.2",
+    "vite": "^5.3.1",
```

Comment:
why?

Reply:
To be frank I don't know, I followed a "getting started with Vue tutorial" and copy-pasted the dependencies. I will try to get rid of it

Reply:
I removed it without issue